enterprise_access/utils.py

```diff
@@ -157,3 +164,33 @@ def get_automatic_expiration_date_and_reason(
         action_required_by['reason'],
     )
     return action_required_by
+
+
+def should_send_email_to_pecu(recent_action):
+    current_date = datetime.today().replace(hour=0, minute=0, second=0, microsecond=0)
+    is_5_days_since_invited = current_date - timedelta(
+        days=BRAZE_GROUPS_EMAIL_CAMPAIGNS_FIRST_REMINDER_DAY
+    ) == (datetime.strptime(recent_action, "%B %d, %Y"))
+    is_25_days_since_invited = current_date - timedelta(
+        days=BRAZE_GROUPS_EMAIL_CAMPAIGNS_SECOND_REMINDER_DAY
+    ) == (datetime.strptime(recent_action, "%B %d, %Y"))
+    is_50_days_since_invited = current_date - timedelta(
+        days=BRAZE_GROUPS_EMAIL_CAMPAIGNS_THIRD_REMINDER_DAY
+    ) == (datetime.strptime(recent_action, "%B %d, %Y"))
+    is_65_days_since_invited = current_date - timedelta(
+        days=BRAZE_GROUPS_EMAIL_CAMPAIGNS_FOURTH_REMINDER_DAY
+    ) == (datetime.strptime(recent_action, "%B %d, %Y"))
+    is_85_days_since_invited = current_date - timedelta(
+        days=BRAZE_GROUPS_EMAIL_CAMPAIGNS_FINAL_REMINDER_DAY
+    ) == (datetime.strptime(recent_action, "%B %d, %Y"))
+
+    if (
+        is_5_days_since_invited or
+        is_25_days_since_invited or
+        is_50_days_since_invited or
+        is_65_days_since_invited or
+        is_85_days_since_invited
+    ):
+        return True
+
+    return False
```

Comment:
nit: Can we simplify this check?
```suggestion
        return is_5_days_since_invited
```